### ISSUE TITLE
Initial visibilty of NavigationDrawer

### ISSUE THREAD

**Issue by trixon** [dlsc-software-consulting-gmbh/WorkbenchFX#103]
I noticed that workbench.getNavigationDrawer().isVisible() returns true from start when it's not visible.
The following code only works after I toggle the NavigationDrawer by mouse
```
if (mWorkbench.getNavigationDrawer().isVisible()) {
    mWorkbench.hideNavigationDrawer();
} else {
    mWorkbench.showNavigationDrawer();
}
```
...unless I first do
```
mWorkbench.getNavigationDrawer().setVisible(false);
```

Should this be done in the framework?

**Comment by dlemmermann** [comment]
yes, it should ... PR? :-)

**Comment by dlemmermann** [comment]
I added it.

**Comment by dlemmermann** [comment]
This will be available in release 11.3.1.

**Comment by trixon** [comment]
Thanks, I'm not really used/comfortable with PR. I guess I should become that. :)
